src/runtime/components/forms/Form.vue

```diff
@@ -264,7 +267,7 @@ async function getValibotError (
   state: any,
   schema: ValibotObjectSchema<any>
 ): Promise<FormError[]> {
-  const result = await schema._parse(state)
+  const result = await ValibotParseAsync(schema, state)
```

Comment:
I believe this should do the trick, imported only at runtime

```suggestion
  const { safeParseAsync } = await import('valibot')
  const result = await safeParseAsync(schema, state)
```